.github/linters/.cspell.json

```diff
@@ -0,0 +1,146 @@
+{
+    "ignorePaths": [
+        "**/node_modules/**",
+        "**/vscode-extension/**",
+        "**/.git/**",
+        "**/.pnpm-lock.json",
+        ".vscode",
+        "megalinter",
+        "package-lock.json",
+        "report"
```

Comment:
**suggestion:** maybe couples lines are not needed here
and we could add `yarn.lock` and `CHANGELOG.md`?

```suggestion
	"CHANGELOG.md",
        "**/node_modules/**",
        "**/.git/**",
        ".vscode",
        "megalinter",
        "yarn.lock",
        "report"
```